package.json

```diff
@@ -111,37 +111,37 @@
     "browserslist": "^4.12.0",
     "chalk": "^4.0.0",
     "enquirer": "^2.3.5",
-    "eslint": "^6.8.0",
+    "eslint": "^7.2.0",
     "eslint-config-airbnb": "^18.1.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-es5": "^1.5.0",
-    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-import": "^2.21.1",
     "eslint-plugin-json": "^2.1.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-react": "^7.20.0",
-    "gh-pages": "2.2.0",
+    "gh-pages": "3.0.0",
```

Comment:
Do we know if there are any breaking changes in this major version bump?

Reply:
no breaking changes, only says that tests will no longer run on Node < 10 ([full updates here](https://github.com/tschaub/gh-pages/releases/tag/v3.0.0))

Reply:
brill, thanks for checking :)